developing_features/guides/new_project.md

```diff
@@ -0,0 +1,79 @@
+# [Developing Features](../developing_features.md) / Guides / Setting Up a New Project
+
+
+#### Checklist
+
+###### For a Public project
+
+ - Generate a new gem with `bundle gem abc`
+ - Commit it, before changing anything, as `[skip] Initial commit from Bundler 1.16.1 (1m)` (giving the appropriate version of Bundler)
+ - [Create a new repo on GitHub](https://github.com/new) (e.g. `cph/abc`)
+ - [Enable branch protection](https://help.github.com/articles/configuring-protected-branches/) on `master` and turn on these settings:
+    - Require pull request reviews before merging
+    - Require status checks to pass before merging
+       - Require branches to be up to date before merging
+       - Select Travis CI - Pull Request
+       > You won't be able to do this until you've pushed commits and had one build run on Travis.
+
+###### For a Private project
+
+ - Generate a new Rails project with `rails new xyz`
+ - Commit it, before changing anything, as `[skip] Initial commit from Rails 5.2.2 (1m)` (giving the appropriate version of Rails)
+ - [Create a new repo on GitHub](https://github.com/new) (e.g. `cph/xyz`)
+    - Configure **Collaborators and Teams**
+       - **Developers** have **Write** access.
+       - **Houston** has **Admin** access.
+       - **Designers** have **Write** access.
+       - **Testers** have **Write** access.
+ - [Create a new app on Errbit](https://errbit.cphepdev.com/apps/new) (You may have to log out and log back in as an Admin)
+     - Fill in **GitHub Repo** (`cph/xyz`) so that lines in the backtraces can be linked to source
+     - Under **Watchers**, add the non-admin users in Errbit
+     - Under **Notification Service**, pick **Webhook** and fill in the URL `https://houst.in/projects/xyz/hooks/exception_report`
+ - Create a new project in Houston [for the appropriate Team](https://houst.in/)
+     - Give it the same slug as its GitHub repo
+     - Set **Version Control** to **Git** and fill in the the URL `git@github.com:cph/xys.git`
+     - Set **Error Tracker** to **Errbit** and fill in the AppId (from the URL `https://errbit.cphepdev.com/apps/:app_id`)
+     - Set **CI Server** to **Jenkins**
+     - Set **Ticket Tracker** to **Houston**
+     - Enable these features:
+        - **Feedback** (i.e. https://houst.in/feedback/by_project/xyz)
+        - **Goals** (so the new project can be added to Roadmaps)
+        - **Releases** (so that you can create release notes for the project)
+ - [Create a new job in Jenkins](https://ci.cphepdev.com/view/all/newJob)
+     - Give it the same name as its slug in Houston
+     - Copy its configuration from an existing job (e.g. **lsb**)
+     - Change **Respository URL** to the URL `git@github.com:cph/xys.git`
+     - Update the Rails project for Jenkins:
```

Comment:
As I looked at this, I appreciated the specific configuration steps included for setting up a project to use Jenkins, but I couldn't shake the feeling that it didn't really fit with the rest of the document -- it dropped down to too low a level compared to what's around it (and the rest of our desktop procedures). Instead, it seemed like something that could/should be a part of the EP toolchain -- I'd much rather have this knowledge encoded in a script than have to copy/paste from these desktop procedures.

And so I made a PR: https://github.com/cph/ep/pull/163. Assuming that PR looks good, would you be willing to update this part to reference using that particular command of the EP toolchain, with the caveat that perhaps one will have to still tweak the `cibuild` script so that all the correct configuration files are copied, etc.?